src/FSharpPlus/Extensions/Dict.fs

```diff
@@ -73,6 +73,19 @@ module Dict =
             | None    -> ()
         dct :> IDictionary<'Key, 'U>
 
+    /// <summary>Applies given function to each value of the given dictionary.</summary>
+    /// <param name="f">The mapping function.</param>
+    /// <param name="x">The input dictionary.</param>
+    ///
+    /// <returns>Returns dictionary with values x for each dictionary value where the function returns Some(x).</returns>
+    let choose f (x: IDictionary<'Key, 'T>) =
```

Comment:
I think this makes sense from the point of view that dictionaries are simply collections indexed by a key.

Reply:
On one side I like to call it `choose` but I fear the risk of having to deal with an eventual inclusion of a `choosei` function in F# core, called `choose`.

This will need more feedback from users.

So, I would say, if you want to merge it now, please rename them to `chooseValues` so we stay in the safe side. As I said before, we can always move to `choose` if we firmly decide to.